analyzere/base_resources.py

```diff
@@ -48,6 +52,35 @@ def __deepcopy__(self, memo):
             return copy.deepcopy(self.__wrapped__, memo)
         return Reference(self._href)
 
+    def __getattribute__(self, name):
```

Comment:
gross, but if it fixes the problem in the current implementation, I guess it's legit.